jmix-gridexport/gridexport-flowui/src/main/java/io/jmix/gridexportflowui/exporter/entitiesloader/LimitOffsetAllEntitiesLoader.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.gridexportflowui.exporter.entitiesloader;
+
+import io.jmix.core.DataManager;
+import io.jmix.core.LoadContext;
+import io.jmix.core.MetadataTools;
+import io.jmix.core.metamodel.model.MetaClass;
+import io.jmix.flowui.model.CollectionLoader;
+import io.jmix.gridexportflowui.GridExportProperties;
+import io.jmix.gridexportflowui.exporter.EntityExportContext;
+import io.jmix.gridexportflowui.exporter.EntityExporter;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.PlatformTransactionManager;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * This loader implements limit-offset pagination strategy.
+ */
+@Component
+public class LimitOffsetAllEntitiesLoader extends AbstractAllEntitiesLoader implements AllEntitiesLoader {
+
+    public static final String LIMIT_OFFSET_EXPORT_DATA_PROVIDER = "limit-offset";
```

Comment:
Why constant name is "data provider"? Isn't it just a "name" or "type"?
Maybe `LimitOffsetAllEntitiesLoader .PAGINATION_TYPE`?